backbone/src/main/java/org/researchstack/backbone/model/survey/factory/SurveyFactory.java

```diff
@@ -632,12 +639,21 @@ public LoginStep createLoginStep(Context context, ProfileSurveyItem item) {
 
     /**
      * @param item InstructionSurveyItem from JSON
-     * @return valid EmailVerificationStep matching the InstructionSurveyItem
+     * @return valid EmailVerificationSubStep matching the InstructionSurveyItem
      */
-    public EmailVerificationStep createEmailVerificationStep(InstructionSurveyItem item) {
-        EmailVerificationStep step = new EmailVerificationStep(item.identifier, item.title, item.text);
-        fillInstructionStep(step, item);
-        return step;
+    public EmailVerificationStep createEmailVerificationStep(Context context, InstructionSurveyItem item) {
```

Comment:
Nit-pick: change method signature return type to EmailVerificationSubStep to match the change to the comment